components/collapsible/collapsible.vue

```diff
@@ -64,13 +64,20 @@
       tabindex="-1"
       appear
       v-on="$listeners"
-      @after-leave="onLeaveTransitionComplete"
-      @after-enter="onEnterTransitionComplete"
+      @transitionfinished="onTransitionFinished"
     >
-      <!-- @slot Slot for the collapsible element that is expanded by the anchor -->
-      <slot
-        name="content"
-      />
+      <!-- @slot Slot for content that is shown when collapsible is expanded -->
```

Comment:
Put these @slot comments above the `<slot>` rather than the `<template>` or else they don't show up in storybook

Reply:
Done